Gems/Atom/Feature/Common/Code/Source/RayTracing/RayTracingAccelerationStructurePass.cpp

```diff
@@ -195,12 +195,26 @@ namespace AZ
 
         RPI::TimestampResult RayTracingAccelerationStructurePass::GetTimestampResultInternal() const
         {
-            return m_timestampResults.at(RHI::MultiDevice::DefaultDeviceIndex);
+            RPI::TimestampResult result{};
+
+            for (auto& [deviceIndex, timestampResult] : m_timestampResults)
+            {
+                result.Add(timestampResult);
```

Comment:
Can we assume that the timestamps of different devices are compatible?

Reply:
Yes, let's assume that xD